server-side-rendering/src/routes/+page.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+	export let data;
+
+	const { user } = data;
+
+	function getInitials(name: string) {
+		const [first, last] = name.split(' ');
+		if (last) return `${first[0]}${last[0]}`;
+		return `${first[0]}`;
+	}
+</script>
+
+<div class="u-max-width-500 u-width-full-line">
+	<h1 class="heading-level-2 u-margin-block-start-auto">Your account 🎉</h1>
+	<div class="u-margin-block-start-24">
+		<section class="card">
+			<div class="user-profile">
+				<span class="avatar">{getInitials(user.name)}</span>
+				<span class="user-profile-info">
+					<span class="name">{user.email}</span>
+					<div class="interactive-text-output u-padding-inline-0">
+						<span class="text">{user.$id}</span>
+						<div class="u-flex u-cross-child-start u-gap-8">
+							<button class="interactive-text-output-button" aria-label="copy text">
+								<span class="icon-duplicate" aria-hidden="true" />
+							</button>
+						</div>
+					</div>
+				</span>
+				<span class="user-profile-sep" />
+				<span class="user-profile-empty-column" />
+				<span class="user-profile-info"><span class="text">Welcome back, {user.name}!</span></span>
```

Comment:
Hmm, this is page of authenticated user but its in main page.sveltte of `/`. Can we instead redirect user and have authenticated page in something like `/dashboard`? That way `/` path will always serve just as a gateway to either login or dashboard. No real page

Reply:
I think thats good use-case, as on `/` you usually have landing page or something - not related to app itself

Reply:
It would make sense with my comment above, when we have 2 folders - one for guests, one for users. That becomes beneficial setup as adding new pages gets easier

Reply:
Went with /account and /signin,/signup